html/dom/interfaces.worker.js

```diff
@@ -5,9 +5,10 @@ importScripts("/resources/WebIDLParser.js", "/resources/idlharness.js");
 
 idl_test(
   ["html"],
-  ["dom", "cssom", "touch-events", "uievents"],
```

Comment:
The other dependencies here didn't make sense and are probably copypasta. Let's see if it affect the results in any way, I expect it won't.

Reply:
Nope, that actually caused a regression in the results: `FAIL message: promise_test: Unhandled rejection with value: "HTMLLinkElement includes LinkStyle, but LinkStyle is undefined."`

I guess we don't change the rules for what dependencies we need based on what the global scope is, and perhaps we shouldn't so that we can test that things *aren't* exposed... not sure, but will revert changes.